### PR TITLE
Clean some words.

### LINKED ISSUE #23: Typos in text.
Two of them:

"but you would normally be accepted to write "

accepted should be expected

"Note the omission of the 'function' keyword as in ES6 it is entirely optional Default attributes for the todo."

Seems to be a word or two missing here.

### PR DESCRIPTION
**Pull request by stephenplusplus**
I wanted to just make it a little easier to read, highlighting the ES6 stuff.

http://stephenplusplus.github.io/traceur-todomvc/docs/app.html for a preview!

(Fixes #23)

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -142,37 +164,36 @@ module TodoApp {
...
+		// *Todos are sorted by their original insertion order.*
 		comparator(todo) {
 			return todo.get('order');
 		}
 	}
 
-	// Create our global collection of **Todos**.
-	var Todos = new TodoList();
+	// *Create our global collection of **Todos**.*
+	let Todos = new TodoList();
```
Comment (stephenplusplus) [inline js/app.js:174]: Right?

File: js/app.js [hunk 2]
```diff
@@ -202,39 +223,41 @@ module TodoApp {
...
 		// post-process step, validating and transforming during assignment.
-		// In general this means using `set` and `get` to bind a property
+		//
+		// In general, this means using `set` and `get` to bind a property
 		// of an object to a function which is invoked when the property is
 		// being set and looked up. [Read more](http://ariya.ofilabs.com/2013/03/es6-and-method-definitions.html)
 		// on getters and setters.
 		get isHidden() {
-			var isCompleted = this.model.get('completed');
+			const isCompleted = this.model.get('completed');
```
Comment (stephenplusplus) [inline js/app.js:238]: Right? :dancer:

File: js/app.js [hunk 3]
```diff
@@ -202,39 +223,41 @@ module TodoApp {
...
+		// *Toggle the `'completed'` state of the model.*
 		toggleCompleted() {
 			this.model.toggle();
 		}
 
-		// Switch this view into `'editing'` mode, displaying the input field.
+		// *Switch this view into `'editing'` mode, displaying the input field.*
 		edit() {
-			var value = this.input.val();
+			const value = this.input.val();
```
Comment (stephenplusplus) [inline js/app.js:252]: Yeah? :dancers:

File: js/app.js [hunk 4]
```diff
@@ -202,39 +223,41 @@ module TodoApp {
...
 
 			this.$el.addClass('editing');
 			this.input.val(value).focus();
 		}
 
-		// Close the `'editing'` mode, saving changes to the todo.
+		// *Close the `'editing'` mode, saving changes to the todo.*
 		close() {
-			var title = this.input.val();
+			const title = this.input.val();
```
Comment (stephenplusplus) [inline js/app.js:260]: Yay? :+1: :-1:

File: docs/app.html [hunk 5]
```diff
@@ -57,7 +57,7 @@
               <p>This is a re-write of the Backbone <a href="http://todomvc.com">TodoMVC</a> app using
 ECMAScript 6 features. It&#39;s made possible using
 <a href="https://github.com/google/traceur-compiler">Traceur</a> compiler and was
-authored by Addy Osmani, Pascal Hartig, Sindre Sorhus, Stephen Sawchuck,
+authored by Addy Osmani, Pascal Hartig, Sindre Sorhus, Stephen Sawchuk,
```
Comment (sindresorhus) [inline docs/app.html:60]: keep it, i prefer Sawchuck :p

Comment (passy) [inline docs/app.html:60]: Yeah, :-1: ;)

File: js/app.js (hunk 1 above)
Comment (passy) [inline js/app.js:174]: Couldn't this be `const`, as well?

File: js/app.js (hunk 2 above)
Comment (passy) [inline js/app.js:238]: :+1:

File: js/app.js (hunk 3 above)
Comment (passy) [inline js/app.js:252]: :cake:

File: js/app.js (hunk 4 above)
Comment (passy) [inline js/app.js:260]: :palm_tree:

File: js/app.js (hunk 1 above)
Comment (sindresorhus) [inline js/app.js:174]: nope, it needs to be mutated.

File: js/app.js (hunk 2 above)
Comment (sindresorhus) [inline js/app.js:238]: const all the things

File: js/app.js (hunk 1 above)
Comment (passy) [inline js/app.js:174]: They way I understand `const` is that it doesn't protect against mutation, but only against re-assignment.

**Pull request merged**